actors/user/src/test/java/org/sunbird/user/UserStatusActorTest.java

```diff
@@ -0,0 +1,150 @@
+import org.keycloak.representations.idm.UserRepresentation;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.KeyCloakConnectionProvider;
+import org.sunbird.common.request.Request;
+import org.sunbird.common.responsecode.ResponseCode;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.models.user.User;
+import org.sunbird.user.actors.UserStatusActor;
+import org.sunbird.user.service.UserService;
+import org.sunbird.user.service.impl.UserServiceImpl;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({UserServiceImpl.class, KeyCloakConnectionProvider.class, ServiceFactory.class})
+@PowerMockIgnore("javax.management.*")
+public class UserStatusActorTest {
+
+  private static final Props props = Props.create(UserStatusActor.class);
+  private static final ActorSystem system = ActorSystem.create("system");
+  private static final String userId = "someUserId";
+  private User user;
+  private static CassandraOperationImpl cassandraOperation;
+
+  @BeforeClass
+  public static void beforeClass() {
+
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    Response response = createCassandraUpdateSuccessResponse();
+    when(cassandraOperation.updateRecord(
+            Mockito.anyString(), Mockito.anyString(), Mockito.anyMap()))
+        .thenReturn(response);
+  }
+
+  @Before
+  public void init() {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    UserRepresentation userRepresentation = mock(UserRepresentation.class);
+    RealmResource realmResource = mock(RealmResource.class);
+
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    Keycloak keycloak = mock(Keycloak.class);
+
+    PowerMockito.mockStatic(UserServiceImpl.class);
+    UserService userService = mock(UserService.class);
+    when(UserServiceImpl.getInstance()).thenReturn(userService);
+
+    PowerMockito.mockStatic(KeyCloakConnectionProvider.class);
+    when(KeyCloakConnectionProvider.getConnection()).thenReturn(keycloak);
+    when(keycloak.realm(Mockito.anyString())).thenReturn(realmResource);
+
+    UsersResource usersResource = mock(UsersResource.class);
+    when(realmResource.users()).thenReturn(usersResource);
+
+    UserResource userResource = mock(UserResource.class);
+    when(usersResource.get(Mockito.any())).thenReturn(userResource);
+    when(userResource.toRepresentation()).thenReturn(userRepresentation);
+    userRepresentation.setEnabled(Mockito.anyBoolean());
```

Comment:
this is not required - if function does not return any value -> mocking is not requred. 
in this case - even mocking is misisng